src/main/res/values/strings.xml

```diff
@@ -181,10 +181,10 @@
     </string-array>
 
     <!-- Main activity -->
-    <string name="ma_title_rate_us">Rate us!</string>
+    <string name="ma_title_rate_us">Rate us.</string>
     <string name="ma_title_add_account">Add an account</string>
     <string name="ma_title_welcome">Welcome</string>
-    <string name="ma_content_welcome">Welcome to Nextcloud SMS application. This application let you synchronize your SMS to your Nextcloud instance using SMS app.</string>
```

Comment:
Says Welcome twice (?) (The string preceeding it is "Welcome".)
Needs "the" Nextcloud SMS application.
Says "Application" twice.
Should be "lets".
Need to qualify "SMS" with messages.
Using "an"(?) SMS app.